documentation/src/main.ts

```diff
@@ -10,13 +10,10 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-import loadPolyfills from '@open-wc/polyfills-loader';
 import polyfill from 'element-closest';
 
-loadPolyfills().then(() => {
```

Comment:
OpenWC no longer suggests the use of this technique https://www.npmjs.com/package/@open-wc/polyfills-loader